deployment/network-operator/values.yaml

```diff
@@ -29,6 +29,9 @@ sriovNetworkOperator:
   # -- Deploy SR-IOV Network Operator.
   enabled: false
 
+nicConfigurationOperator:
```

Comment:
This should be done in the template `/hack/templates/values/values.template`

Reply:
done